src/discretediag.jl

```diff
@@ -27,9 +25,9 @@ function hangartner_inner(Y::AbstractMatrix, m::Int)
 end
 
 """
-  weiss(X::AbstractMatrix) -> (statistic, m_tot, pvalue, ca)
+    weiss(X::AbstractMatrix)
 
-The weiss procedure to assess convergence in MCMC output computes X^2/c and evaluates a p-value from the X^2 distribution with (|R| − 1)(s − 1) degrees of freedom.
+The weiss procedure to assess convergence in MCMC output computes ``\\frac{X^2}{c}`` and evaluates a p-value from the ``X^2`` (Chi-square) distribution with ``(|R| − 1)(s − 1)`` degrees of freedom.
```

Comment:
Slightly better maybe:
```suggestion
Assess the convergence of the MCMC chains with the Weiss procedure.

It computes ``\\frac{X^2}{c}`` and evaluates a p-value from the ``\\chi^2`` distribution with ``(|R| − 1)(s − 1)`` degrees of freedom.
```

However, it would be good to add a reference and explain all variables.